tests/unittests/OperatorTest.cpp

```diff
@@ -1282,6 +1285,43 @@ TEST_P(InterpAndCPU, concatVectors) {
   }
 }
 
+TEST_P(InterpAndCPU, concatVectorsRepeated) {
+  F_->setName("concatVectors");
```

Comment:
Can you add some comments on the test? What does it check and how.

Reply:
Done.